api/src/main/java/io/opencensus/metrics/MetricOptions.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.metrics;
+
+import com.google.auto.value.AutoValue;
+import io.opencensus.internal.Utils;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Options for every metric added to the {@link MetricRegistry}.
+ *
+ * @since 0.20
+ */
+@Immutable
+@AutoValue
+public abstract class MetricOptions {
+
+  /**
+   * Returns the description of the Metric.
+   *
+   * <p>Default value is {@code ""}.
+   *
+   * @return the description of the Metric.
+   */
+  public abstract String getDescription();
+
+  /**
+   * Returns the unit of the Metric.
+   *
+   * <p>Default value is {@code "1"}.
+   *
+   * @return the unit of the Metric.
+   */
+  public abstract String getUnit();
+
+  /**
+   * Returns the list of label keys for the Metric.
+   *
+   * <p>Default value is {@link Collections#emptyList()}.
+   *
+   * @return the list of label keys for the Metric.
+   */
+  public abstract List<LabelKey> getLabelKeys();
+
+  /**
+   * Returns the map of constant labels (they will be added to all the TimeSeries) for the Metric.
+   *
+   * <p>Default value is {@link Collections#emptyMap()}.
+   *
+   * @return the list of label keys for the Metric.
```

Comment:
`s/the list of label/the map of label/`?

Reply:
done.